src/ripple/rpc/handlers/LedgerData.cpp

```diff
@@ -78,28 +79,29 @@ Json::Value doLedgerData (RPC::Context& context)
     jvResult[jss::ledger_hash] = to_string (lpLedger->info().hash);
     jvResult[jss::ledger_index] = std::to_string(lpLedger->info().seq);
```

Comment:
we're inconsistent on this in the codebase!  I'm going to stick with what's there for now.